README.md

```diff
@@ -80,6 +84,13 @@ Scuba defines the following environment variables in the container:
   where `.scuba.yml` was found.
 
 
+## Bash Completion
+Scuba supports command-line completion using the [`argcomplete` package](https://github.com/kislyuk/argcomplete).  Per
+the [`argcomplete` README](https://github.com/kislyuk/argcomplete#global-completion), command-line completion can be
+activated by running `eval "$(register-python-argcomplete scuba)"` manually *or* by using the
+`activate-global-python-argcomplete` script.
```

Comment:
It's not really scuba's responsibility to teach users how to use `argcomplete`, but I think we could help the user a little more here.

- [ ] Running `eval "$(register-python-argcomplete scuba)` will only enable completion for the current instance of the shell. What a user would *really* want to do is add that line to either `.bashrc`, or perhaps more appropriately, `~/.bash_completion`.
- [ ] Running `activate-global-python-argcomplete` as a normal user will fail, because it tries to write to `/etc/bash_completion.d/` by default.
- [ ] Running `activate-global-python-argcomplete --user` is probably better, but by itself is incomplete.
  - It will install the script as `~/.bash_completion.d/python-argcomplete`
  - `~/.bash_completion.d/*` are *not* sourced by default. However `~/.bash_completion` is, so the [solution](https://serverfault.com/a/831184) is to add this to `~/.bash_completion`:
    ```bash
    for bcfile in ~/.bash_completion.d/* ; do
      [ -f "$bcfile" ] && . $bcfile
    done
    ```

Reply:
Okay.  I'll update this with some more explicit `argcomplete` activation examples.

Reply:
Addressed by TimothySprague/scuba@ea1be7f6fb2a25a586329f831373ff28e7a0a2db